composer/lib/dependabot/composer/native_helpers.rb

```diff
@@ -8,7 +8,7 @@ def self.composer_helper_path(composer_version: "v2")
       end
 
       def self.composer_helpers_dir
-        File.join(native_helpers_root, "composer/helpers")
+        File.join(native_helpers_root, "composer")
```

Comment:
Looking at this with fresh eyes, perhaps we should modify this to check if `composer/vX` folders exist and if not fall back to the original path to maintain compatibility between image and gem for a few releases?

Reply:
I'm mostly OK with ensuring we deploy this side by side, our process is set up to do so any way for this exact reason.

We can bump the minor version and mention it's a breaking change in the CHANGELOG?